package avalon.network;

message ChatMessage {
    required int32 sender_id = 1;
    required string message_text = 2;
    required int32 timestamp = 3;
}
